tests/env/test_explicit_environment.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+"""Tests for Environment class with explicit functionality."""
+
+import pytest
+
+from conda.env.env import Environment
+from conda.env.specs.requirements import RequirementsSpec
+from tests.env import support_file
+
+
+@pytest.fixture(scope="module")
+def support_explicit_file():
+    """Path to the explicit environment file in the test support directory"""
+    return support_file("explicit.txt")
+
+
+@pytest.fixture(scope="module")
+def explicit_urls():
+    """Sample URLs for testing explicit environments."""
+    return [
+        "https://conda.anaconda.org/conda-forge/linux-64/numpy-1.21.5-py39h2a9ead8_0.tar.bz2",
+        "https://conda.anaconda.org/conda-forge/linux-64/python-3.9.10-h85951f9_5_cpython.tar.bz2",
+    ]
+
+
+@pytest.fixture(scope="function")
+def explicit_env(explicit_urls):
+    """Create an Environment instance with explicit specs for testing."""
+    return Environment(
+        name="test-explicit",
+        dependencies=["@EXPLICIT"] + explicit_urls,
+        filename="/path/to/explicit-file.txt",
+    )
+
+
+def test_explicit_environment_is_explicit():
+    """Test that Environment with @EXPLICIT marker returns True for explicit()."""
+    env = Environment(dependencies=["@EXPLICIT", "test"])
+    assert env.dependencies.explicit
+
+    regular_env = Environment()
+    assert not regular_env.dependencies.explicit
+
+
+def test_explicit_environment_initialization(explicit_urls):
+    """Test Environment initialization with explicit parameters."""
+    explicit_deps = ["@EXPLICIT"] + explicit_urls
```

Comment:
Can we add a test where `@EXPLICIT` is not the first line, just to make sure it works elsewhere in the file?